routers/routes/routes.go

```diff
@@ -281,6 +281,7 @@ func RegisterRoutes(m *macaron.Macaron) {
 		m.Group("/repos", func() {
 			m.Get("", admin.Repos)
 			m.Post("/delete", admin.DeleteRepo)
+			m.Post("/:repoid", admin.GetRepoName)
```

Comment:
Why Post, for a GetSomething?

Reply:
yes, i create new route in 'routes.go' and function in 'repos.go'. that code use to get repository name by id. the result, repository name will be display in delete repository message